receiver/kafkametricsreceiver/README.md

```diff
@@ -12,14 +12,17 @@ Required settings (no defaults):
     - `topics`
     - `consumers`
     - `brokers`
+    
+Metrics collected by the associated scraper are listed [here](metadata.yaml)
 
 Optional Settings (with defaults):
 
 - `brokers` (default = localhost:9092): the list of brokers to read from.
-- `topic_match` (default = *): regex pattern of topics to filter for metrics collection.
-- `group_match` (default = *): regex pattern of consumer groups to filter on for metrics.
+- `topic_match` (default = ^[^_].*$): regex pattern of topics to filter on metrics collection. The default filter excludes internal topics (starting with `_`).
+- `group_match` (default = .*): regex pattern of consumer groups to filter on for metrics.
 - `client_id` (default = otel-metrics-receiver): consumer client id
 - `collection_interval` (default = 1m): frequency of metric collection/scraping.
+- `labels` (default none): list of additional labels to be applied to metrics. This should be in `key:value` format.
```

Comment:
Generally this kind of generic operation should be done in a processor, e.g. metricstransformprocessor.

Reply:
oh wow, I completely missed this. I will remove the custom labels then.